README.md

```diff
@@ -1,60 +1,92 @@
-# Source Code Annotation application
+[![Build Status](https://travis-ci.org/src-d/code-annotation.svg)](https://travis-ci.org/src-d/code-annotation)
+![unstable](https://svg-badge.appspot.com/badge/stability/unstable?a)
+
+# Source Code Annotation Tool
 
 In order to evaluate quality of ML models, as well as to create “ImageNet for source core” there is a need for tools to automate the data collection/labeling/annotation.
 
+![Screenshot](.github/screenshot.png?raw=true)
```

Comment:
I'm not sure what this screenshot shows.
Does it detect there's a new line of code? Is that line relevant?

Reply:
it shows the interface of the tool. There are 2 files, the difference is in 1 line. We ask a user to mark it as identical/similar/different. It is what this tool does.